components/meter/meter-radial.js

```diff
@@ -0,0 +1,56 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { heading4Styles } from '../typography/styles.js';
+
+class MeterRadial extends LitElement {
+	static get properties() {
+		return {
+			value: { type: Number },
+			max: { type: Number }
+		};
+	}
+
+	static get styles() {
+		return [ heading4Styles, css`
+		:host {
+			display: inline-block;
+			width: 86px;
+		}
+		.d2l-meter-radial-full-bar,
+		.d2l-meter-radial-progress-bar {
+			fill: none;
+			stroke-linecap: round;
+			stroke-width: 9;
+		}
+		.d2l-meter-radial-full-bar {
+			stroke: var(--d2l-color-gypsum);
+		}
+		.d2l-meter-radial-progress-bar {
+			stroke: var(--d2l-color-celestine);
+		}
+		.d2l-meter-radial-text {
+			color: var(--d2l-color-ferrite);
+		}
+	` ];
+	}
+
+	render() {
+		const lengthOfLine = 115; // found by approximating half the perimeter of the ellipse with radii 38 and 35
+		const progressFill = this.value / this.max * lengthOfLine;
+		return html `
+			<svg viewBox="0 0 86 52" >
+				<g transform="translate(5 47)">
```

Comment:
If you paste this into SVGOMG, it should remove the `<g>` with the transform and apply the changes down to the paths. Then you can bring it back here without the`<g>`.

Reply:
Whoops missed that step. So interesting, I did at first do the math on my own to move everything and the `text` didn't go exactly where I wanted it to. I need more understanding on how the position of text fields work. I know there is a relative one you can use. 

Anyways, basically, for the `text` field SVGOMG put the transform down. 

I find moving the origin easier to read vs adjusting the all the elements inside to fit a bad origin. I'll push the new version up.